Rakefile

```diff
@@ -78,6 +38,25 @@ task :default => 'test'
 
 private
 
+def run_tests(scheme, sdk, destination)
+  sim_destination = "-destination \'#{destination}\'"
+  sh("xcodebuild -workspace PubNub.xcworkspace -scheme '#{scheme}' -sdk '#{sdk}' #{sim_destination} -configuration 'Debug' clean test | xcpretty -c ; exit ${PIPESTATUS[0]}") rescue nil
```

Comment:
That's interesting. I think I use the Debug flag for testing. Let me check for the effect of that. Shouldn't be too hard.